components/Modal/Modal.jsx

```diff
@@ -120,12 +120,18 @@ class Modal extends React.Component {
   componentWillUnmount() {
     const { body } = document;
     this.focusedElementBeforeOpen.focus();
+    this.setBodyOverflow('auto');
 
     body.removeChild(this.modalOverlay);
     window.removeEventListener('keydown', this.handleKeyDown);
     window.removeEventListener('keydown', this.handleEscKey);
   }
 
+  setBodyOverflow = value => {
+    const { body } = document;
+    body.style.overflow = value;
```

Comment:
Using in the Next.js application, does it work?

Reply:
I've tested here and didnt break.
But i implemented this suggestion anyway.
Good point.